core/src/test/scala/org/apache/spark/sql/ViewTestSuite.scala

```diff
@@ -4,23 +4,29 @@ class ViewTestSuite extends BaseTiSparkTest {
   private val table = "test_view"
 
   test("Test View") {
-    tidbStmt.execute(s"drop table if exists $table")
```

Comment:
what if there's some dirty data (e.g. a table named `test_view`) in database?

Reply:
afterAll will drop it.

Reply:
but we should still drop it before each test

Reply:
see this
![image](https://user-images.githubusercontent.com/5574887/67822067-186bd980-fafa-11e9-903e-72e16e7a1502.png)

Reply:
Is it partitionTableSuite? If you take a look at `afterAll`, you will find that table `t` is not dropped at `afterAll`.